_shared/project/Makefile

```diff
@@ -12,6 +12,13 @@ services: python
 	@docker compose $(args)
 {% endif %}
 
+.PHONY: db
+$(call help,make db,initialize the DB and upgrade it to the latest migration)
+db: args?=upgrade head
+db: python
+	@tox -qe dev --run-command 'python bin/make_db'
+	@tox -qe dev  --run-command 'alembic -c conf/alembic.ini $(args)'
```

Comment:
This is the `make db` command exactly as currently found on LMS, Checkmate and h's `main` branches